api/config.py

```diff
@@ -321,3 +324,9 @@ def mongo_pipeline(table, pipeline):
 
 def get_auth(auth_type):
     return get_config()['auth'][auth_type]
+
+
+# Storage configuration
+fs = open_fs(__config['persistent']['fs_url'])
+legacy_fs = open_fs('osfs://' + __config['persistent']['data_path'])
+support_legacy_fs = True
```

Comment:
Can this be toggled without code change/redeploy?
I assumed we want that, and definitely desirable if not too hard.